packages/server/beaver_triple_service/utils/interceptor.go

```diff
@@ -35,33 +35,33 @@ func BtsAuthFunc(ctx context.Context) (context.Context, error) {
 }
 
 func getSecret() ([]byte, error) {
-	raw, ok := os.LookupEnv("JWT_SECRET_KEY")
-	if !ok {
-		return nil, status.Error(codes.Internal, "jwt auth key is not provided")
-	}
-	secret, err := base64.StdEncoding.DecodeString(raw)
-	if err != nil {
-		return nil, err
+	var secrets []byte
+	if os.Getenv("JWT_SECRET_KEY") != "" {
+		secrets = []byte(os.Getenv("JWT_SECRET_KEY"))
+	} else {
+		secrets = []byte("hoge")
 	}
-	return secret, nil
+
+	return secrets, nil
 }
 
 func AuthJWT(tokenString string) (*jwt_types.Claim, error) {
 	jwtSecret, err := getSecret()
 	if err != nil {
 		return nil, err
 	}
+	fmt.Printf("%s", jwtSecret)
```

Comment:
This is debug code?

Reply:
@come2ry please check.

Reply:
https://github.com/acompany-develop/QuickMPC/pull/144#discussion_r1145598281  
already pointed out